src/libraries/Microsoft.Extensions.Hosting/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:Microsoft.Extensions.Hosting.BackgroundServiceExceptionBehavior.#ctor</Target>
```

Comment:
That's one of the compat differences emitted that I absolutely don't understand. Why would an enum ctor be present in one assembly but not the other. I checked that both run with references. Maybe it's the linker that strips it out in one assembly. We would need to check.